packages/atomic/src/components/common/breadbox/breadbox.tsx

```diff
@@ -0,0 +1,241 @@
+import {FacetValueState} from '@coveo/headless';
```

Comment:
This is not something we can do: Importing from `@coveo/headless` in a common component.

I will touch upon this in a separate comment. It's related to the interfaces for functional components, and them doing too much.